runtime/doc/api.txt

```diff
@@ -1805,15 +1805,14 @@ nvim_parse_cmd({str}, {opts})                               *nvim_parse_cmd()*
     Return: ~
         Dictionary containing command information, with these keys:
         • cmd: (string) Command name.
-        • range: (array) Command <range>. Can have 0-2 elements depending on
```

Comment:
Using `<range>` is a bit misleading here as `<range>` is the number of items in the range.

Reply:
Should just change it to "range" then